core/tests/protractor_desktop/skillEditor.js

```diff
@@ -46,7 +46,9 @@ describe('Skill Editor functionality', function() {
     explorationEditorMainTab = explorationEditorPage.getMainTab();
     users.createAndLoginAdminUser(
       'creator@skillEditor.com', 'creatorSkillEditor');
-    topicsAndSkillsDashboardPage.get();
+    waitFor.pageToFullyLoad();
+    browser.get('/topics_and_skills_dashboard/');
```

Comment:
Is it possible to click to get there?

Reply:
yes, it is possible. Earlier in that file after changing the role to admin it is directly navigating to the  `/skill_editor/null` and failing the end to end test. I changed that in order to prevent it 
````
 this.get = function(skillId) {
    browser.get(EDITOR_URL_PREFIX + skillId);
    return waitFor.pageToFullyLoad();
  };
````
this function was running with the id null.
should i modify it accordingly to navigate there via the button click

Reply:
yes, please use the button

Reply:
@kevinlee12  can we add it to the linter test list to avoid it in future
Thanks

Reply:
Doesn't the line you removed click a button? If so can you revert this change?

Reply:
> @kevinlee12 can we add it to the linter test list to avoid it in future
> Thanks

There are still cases there it's still very valid to have have a direct link, so a linter wouldn't be appropriate.

Reply:
@nithusha21 
````
this.get = function(skillId) {
    browser.get(EDITOR_URL_PREFIX + skillId);
    return waitFor.pageToFullyLoad();
  };
````
this funtion was running with the null value
should i revert the changes ?

Reply:
@nithusha21 I'll defer this to you.